Extension/package.nls.json

```diff
@@ -32,6 +32,7 @@
     "c_cpp.command.GoToNextDirectiveInGroup.title": "Go to next preprocessor directive in conditional group",
     "c_cpp.command.GoToPrevDirectiveInGroup.title": "Go to previous preprocessor directive in conditional group",
     "c_cpp.command.CreateDeclarationOrDefinition.title": "Create Declaration / Definition",
+    "c_cpp.command.CopyDeclarationOrDefinition.title": "Copy declaration to clipboard",
```

Comment:
Is this string used? Why does it only mention "declaration" and not "definition"?

Reply:
This is currently not user facing in any way but you are correct, the string needs to be changed. If a command is not user facing, should we remove it?